extension-impl/remoting-resteasy/src/main/java/com/alipay/sofa/rpc/server/rest/SofaNettyJaxrsServer.java

```diff
@@ -80,6 +85,13 @@
     protected boolean                  telnet              = true;                        // CHANGE:是否允许telnet
     protected boolean                  daemon              = true;                        // CHANGE:是否守护线程
```

Comment:
If add `serverConfig` as one parameter of constructor, `keepAlive`, `telnet` and `daemon` is no need to keep.